db/schema.rb

```diff
@@ -0,0 +1,32 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2018_12_19_011339) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "mentors", force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.string "city"
+    t.string "state"
+    t.string "slack_username"
+    t.string "linkedin_username"
+    t.string "github_username"
+    t.boolean "matched"
+    t.integer "user_id"
```

Comment:
I recommend removing the user_id attribute until we make the User model. At that time, we will add a foreign key association.

Reply:
And removed.